api/envoy/config/accesslog/v3/accesslog_format.proto

```diff
@@ -0,0 +1,79 @@
+syntax = "proto3";
+
+package envoy.config.accesslog.v3;
+
+import "google/protobuf/struct.proto";
+
+import "udpa/annotations/status.proto";
+
+option java_package = "io.envoyproxy.envoy.config.accesslog.v3";
+option java_outer_classname = "AccesslogFormatProto";
+option java_multiple_files = true;
+option (udpa.annotations.file_status).package_version_status = ACTIVE;
+
+// [#protodoc-title: AccessLog format]
+
+// Configuration to use multiple :ref:`command operators <config_access_log_command_operators>`
+// to form access log data and format.
+message AccessLogFormat {
```

Comment:
This is going to be used in more than access logs, so IMO if we are going to go through the effort to do this we should name it something more general. `FormatString`, `SubstitutionFormatString`, something else? @envoyproxy/api-shepherds?

Reply:
How about OperatorFormatString 

since we call them as [operators](https://www.envoyproxy.io/docs/envoy/latest/configuration/observability/access_log/usage.html?highlight=command%20operator#command-operators)

Reply:
Another suggestion: `StreamFormatString`, since they are generally formatted strings related to attributes of a stream.

Reply:
Still very vague, 

How about CommandOperatorComposition.   Since it has something to do with our command operators. 

I think Nginx has "variable" which is similar to our "command operator".  Should we rename our "command operator" as "variable"?  

If so, we can call this message as  "VariableComposition"

Reply:
For now,  I will call it CommandOperatorFormat and move the proto file to `envoy/config/core/v3`

Reply:
This is one of these naming discussions that is going to probably depend on who you ask. I'm happy to go with the flow, but it would be helpful to have @mattklein123 provide a preference here.

I think "composition" is definitely getting even more confusing TBH though, this is an unfamiliar terminology to most Envoy users.

Reply:
It is called CommandOperatorFormat now

Reply:
If you are asking me to weigh in I would probably say here is my preference in order:
1) CommandOperatorFormatString (it's long but I think most specific and clear)
2) OperatorFormatString (shorter, not quite as specific)
3) SubstituionFormatString (even more general, might be easier to understand)

I think in general though I would keep FormatString. WDYT?

/wait-any

Reply:
OK, I will go SubstituionFormatString.  It is short and easier to understand.   Thanks